### DISCUSSION TITLE
2022.09.24 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#94] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
### 이번 주 목표 및 달성률(2022.09.18 ~ 2022.09.24)
이은혁 : 민코딩 진도 따라가기, velog 글 7개 작성, 자바의 정석 요약본 1회독 하기
고영석 : 민코딩 진도 따라가기, 백준 IM 대비 문 풀기
김지애 : 민코딩 진도 따라가기, 과거 과제 꼭꼭꼭 완료하기, 운동 3회 (만약 여유가 된다면? 1일 1백준)
김지오 : SWEA IM 대비문제까지 풀기, 헷갈리거나 어려웠던 문제 다 기술블로그에 쓰기
나혜승 : SWEA 1일 두 문제 이상 풀기, 수업 진도 이해하기, 노션 내용 블로그에 옮기기
이수연 : 깃허브 7 커밋 이상하기, 기술블로그 7개 작성, 정처기 필기 인강 1강 이상 듣기
이문삼 : 민코딩 진도 따라가기, 블로그 글 10개 작성, 복싱 2회

**Comment by sooyeonlee127** [comment]
이번주 목표: 깃허브 7 커밋 이상하기, 기술블로그 7개 작성, 정처기 필기 인강 1강 이상 듣기

점수: 30점
깃허브 커밋 4일 했음. 아직도 깃허브가 익숙지 않아서 자주 안쓰게 되는데, 해야하는 거니까 귀찮아도 노력해야겠습니다...
기술블로그 4개 올림. 꾸준하게 해야겠습니다..
정처기 필기 인강 이번주에도 안 들었습니다.....

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 민코딩 진도 따라가기, velog 글 7개 작성, 자바의 정석 요약본 1회독 하기

점수 : 20점
이유 : 
- 민코딩 진도 따라가기 succeeded
- velog 글 7개 작성 failed
- 자바의 정석 요약본 1회독 하기 failed

이번주 velog는 아직 하나도 못올렸습니다. 오늘중으로 노션에 정리해둔거 올리도록 하겠습니다. 
[인증](https://velog.io/@dmsgur7112/series)
자바는 계속 미루고 미루는 중입니다. 요약본 1회독은 커녕 이제 8페이지 읽었습니다. 반성하겠습니다.
그나마 민코딩 진도는 겨우 따라가는데 성공했으니,,, 저한테 20점 부여하겠읍니다.

**Comment by giokim12** [comment]
김지오 : SWEA IM 대비문제까지 풀기, 헷갈리거나 어려웠던 문제 다 기술블로그에 쓰기
점수: 75점
SWEA 시험범위 부분만 풀고나서 백준으로 넘어가부렀습니다
기술블로그에 헷갈렸던거는 다 썼어요. 9월 블로그 포스팅 30개... 고지가 보인닷

**Comment by moonthree** [comment]
이번주 목표 : 민코딩 진도 따라가기, 블로그 글 10개 작성, 복싱 2회
점수 : 60점
민코딩 진도 따라가기와 복싱 2회 출석에 성공했습니다.
하지만 블로그 글을 8개만 올렸고.. 어제 SWEA를 거의 못풀어서 60점입니다.

**Comment by YoungSeok222222** [comment]
이번주 목표: 민코딩 진도 따라가기, 백준 IM 대비 문제 풀기
점수: 45점
이유: 민코딩 진도는 42.5빼고는 다 따라잡았지만, 백준 IM 문제는 많이 못풀었기 때문에 45점을 줬습니다.

**Comment by HyeseungNA** [comment]
이번주 목표 : SWEA 1일 두 문제 이상 풀기, 수업 진도 이해하기, 노션 내용 블로그에 옮기기
점수 : 30점
알고리즘 문제는 다 풀었습니다.
수업 진도는 이해했으나 복습을 하지 못해서..다 잊어버린것 같습니다.
새로운 블로그를 만들지 않아서 일단 노션에 정리하고 있습니다.
